app/src/main/java/com/blink22/android/gitdb/GitContributorsListActivity.java

```diff
@@ -0,0 +1,47 @@
+package com.blink22.android.gitdb;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+
+import com.f2prateek.dart.Dart;
+import com.f2prateek.dart.InjectExtra;
+
+public class GitContributorsListActivity extends AppCompatActivity {
```

Comment:
Why not extending the SignleFragmentActivity class? I see you're doing the same fragment stuff in the onCreate method. It'd be nice not to repeat these boilerplates.